buildkit/build.go

```diff
@@ -47,12 +47,17 @@ func BuildWithBuildkitClient(appDir string, plan *plan.BuildPlan, opts BuildWith
 
 	buildkitHost := os.Getenv("BUILDKIT_HOST")
 	if buildkitHost == "" {
-		log.Error("BUILDKIT_HOST environment variable is not set")
-		return fmt.Errorf("BUILDKIT_HOST environment variable is not set")
+		return fmt.Errorf(`BUILDKIT_HOST environment variable is not set.
```

Comment:
multiline error messages do not seem like idiomatic go, but I don't see a better mechanism to do this?

Maybe better to use the `cxt.logger.LogError` here?